packages/core/src/fhir-to-cda/cda-templates/clinical-document/clinical-document.ts

```diff
@@ -55,16 +67,19 @@ export function buildClinicalDocumentXml(
       ),
       id: buildInstanceIdentifier({
         assigningAuthorityName: clinicalDocumentConstants.assigningAuthorityName,
-        root: clinicalDocumentConstants.idRoot,
+        root: clinicalDocumentConstants.rootOid,
       }),
       code: buildCodeCe({
-        code: clinicalDocumentConstants.code.code,
+        code: "NOTE-TYPE", // TODO: Make this dynamic. IMPORTANT
         codeSystem: clinicalDocumentConstants.code.codeSystem,
         codeSystemName: clinicalDocumentConstants.code.codeSystemName,
-        displayName: clinicalDocumentConstants.code.displayName,
+        displayName: "NOTE-NAME", // TODO: Make this dynamic. IMPORTANT
       }),
-      title: clinicalDocumentConstants.title,
-      effectiveTime: withNullFlavor(clinicalDocumentConstants.effectiveTime, _valueAttribute),
+      title: "NOTE-TITLE", // TODO: Make this dynamic. IMPORTANT
```

Comment:
are all these TODOs not doable right now?

Reply:
Fixed. But there's still room for improvement, so I added to the [improvements ticket](https://github.com/metriport/metriport-internal/issues/1796)